File: backoffice/FondaBackEnd/Exceptions/FindFavoriteRestaurantFondaWebApiControllerException.cs
using System;

namespace com.ds201625.fonda.FondaBackEnd.Exceptions
{
    /// <summary>
    /// Clase FindFavoriteRestaurantFondaWebApiControllerException
    /// </summary>
    public class FindFavoriteRestaurantFondaWebApiControllerException : FondaBackendException
    {
        public FindFavoriteRestaurantFondaWebApiControllerException () : base() {	}

		public FindFavoriteRestaurantFondaWebApiControllerException (string message) : base(message) {	}

        public FindFavoriteRestaurantFondaWebApiControllerException(string message, Exception InnerException)
			: base(message, InnerException) {	}
    }
}

File: frontoffice/app/src/main/java/com/ds201625/fonda/data_access/retrofit_client/exceptions/FindFavoriteRestaurantFondaWebApiControllerException.java
package com.ds201625.fonda.data_access.retrofit_client.exceptions;

import com.ds201625.fonda.domains.FondaBaseException;

/**
 * FindFavoriteRestaurantFondaWebApiControllerException
 * clase de excepcion de tipo web api controller
 */
public class FindFavoriteRestaurantFondaWebApiControllerException extends FondaBaseException {

    public FindFavoriteRestaurantFondaWebApiControllerException() {
    }

    public FindFavoriteRestaurantFondaWebApiControllerException(String detailMessage) {
        super(detailMessage);
    }

    public FindFavoriteRestaurantFondaWebApiControllerException
                                            (String detailMessage, Throwable throwable) {
        super(detailMessage, throwable);
    }

    public FindFavoriteRestaurantFondaWebApiControllerException(Throwable throwable) {
        super(throwable);
    }

}
